rabbitmq/src/urabbitmq/impl/amqp_channel.cpp

```diff
@@ -67,6 +67,7 @@ AMQP::Table CreateHeaders() {
 
   AMQP::Table headers;
   headers["u-trace-id"] = span->GetTraceId();
+  headers["u-parent-span-id"] = span->GetParentId();
```

Comment:
Shouldn't this be `span->GetSpanId()` instead, so the span created in consumer is linked to the span where `Publish(...)` happens, not its parent?

Reply:
Oh, sure, sorry.